src/hooks/data-explorer.ts

```diff
@@ -1,9 +1,10 @@
 const columns = ref<any>([])
 
 // todo: change init code
-const code = ref('select * from scripts')
+const code = ref('select * from system_metrics')
```

Comment:
this table is not built-in, it's just from our docs. It's not recommended to make it default.

Reply:
```suggestion
const code = ref('select * from scripts')
```